spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -107,6 +107,48 @@ public class MyClient{
 }
 ----
 
-If an annotation is not present, the configured properties are ignored. 
-For instance, the property `com.acme.ClassA/methodB/Retry/maxRetries` will be ignored if `@Retry` annotation is not specified on the `methodB` of `com.acme.ClassA`. 
+If an annotation is not present, the configured properties are ignored.
+For instance, the property `com.acme.ClassA/methodB/Retry/maxRetries` will be ignored if `@Retry` annotation is not specified on the `methodB` of `com.acme.ClassA`.
 Similarly, the property `com.acme.ClassA/Retry/maxRetries` will be ignored if `@Retry` annotation is not specified on the class `com.acme.ClassA` as a class-level annotation.
+
+=== Disabled Fault Tolerance policies with configuration
+
+Fault Tolerance policies can be disabled with configuration at method level, class level or globally for all deployment.
+
+Each policy can be disabled by using its annotation name.
+
+* Disabling a policy at Method level
+
+A policy can be disabled at method level with the following config property and value:
+
+`<classname>/<methodname>/<annotation>/enabled=false`
+
+For instance the following config will disable circuit breaker policy on `methodA` of `com.acme.test.MyClient` class:
+
+`com.acme.test.MyClient/methodA/CircuitBreaker/enabled=false`
+
+Policy will be disabled even if the policy is also defined at class level
+
+* Disabling a policy at class level
+
+A policy can be disabled at class level with the following config property and value:
+
+`<classname>/<annotation>/enabled=false`
+
+For instance the following config will disable fallback policy on `com.acme.test.MyClient` class:
```

Comment:
Fallback is only a method level annotation. You might choose Retry for this explanation.